app/controllers/concerns/curation_concerns/collections_controller_behavior.rb

```diff
@@ -13,6 +13,9 @@ module CollectionsControllerBehavior
 
       # This is needed as of BL 3.7
       copy_blacklight_config_from(::CatalogController)
+      configure_blacklight do |config|
+        config.search_builder_class = CurationConcerns::CollectionSearchBuilder
```

Comment:
If you want to include "Curations Concerns Collections Controller Behavior":
1. it should include a default `search_builder_class`
2. this is the only logical default value